iCubTelemVizServer/devices/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2021 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the Apache-2.0 license
+
+cmake_minimum_required(VERSION 3.5)
+project(matlab-whole-body-simulator)
```

Comment:
Copy paste

Reply:
oups. Well, actually these two lines should have been removed, as I moved them to the main CMakeLists.txt.

Reply:
Done.